modular_skyrat/modules/customization/modules/mob/living/carbon/carbon_update_icons.dm

```diff
@@ -73,11 +73,17 @@
 			var/datum/sprite_accessory/snouts/S = GLOB.sprite_accessories["snout"][dna.species.mutant_bodyparts["snout"][MUTANT_INDEX_NAME]]
 			if(S.use_muzzled_sprites && head.mutant_variants & STYLE_MUZZLE)
 				used_style = STYLE_MUZZLE
+		else if(dna?.species.id == "teshari")
```

Comment:
I feel like that's not the best way to do it, try to use `isteshari(src)` or something like that? Also, putting it on an else if means that if your teshari has a snout, this won't produce the effect you desire.